back/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3.8"
+
+services:
+  db-lounge-rent:
+    image: mysql:latest
+    container_name: db-lounge-rent
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_DATABASE: lounge_rent
```

Comment:
agrega el volumen de inicializacion como :

    volumes:
      - ./db_scripts:/docker-entrypoint-initdb.d
con el archivo con este contenido
`GRANT ALL PRIVILEGES ON *.* TO 'user'@'%';`